tests/scripts/pulpcore/test_task.sh

```diff
@@ -15,23 +15,25 @@ trap cleanup EXIT
 
 sync_task="pulp_file.app.tasks.synchronizing.synchronize"
 expect_succ pulp task list --name $sync_task --state canceled
-#count="$(echo "$OUTPUT" | jq -r length)"
+count="$(echo "$OUTPUT" | jq -r length)"
 
 expect_succ pulp file remote create --name "cli_test_file_remote" \
   --url "$FILE_REMOTE_URL"
 expect_succ pulp file remote create --name "cli_test_file_large_remote" \
   --url "$FILE_LARGE_REMOTE_URL"
 expect_succ pulp file repository create --name "cli_test_file_repository" --remote "cli_test_file_large_remote"
 
-# skip these until https://pulp.plan.io/issues/7980 is resolved
 # Test canceling a task
-#expect_succ pulp --background file repository sync --name "cli_test_file_repository"
-#task="$(echo "$ERROUTPUT" | grep -E -o "/pulp/api/v3/tasks/[-[:xdigit:]]*/")"
-#expect_succ pulp task cancel --href "$task"
-#expect_succ pulp task list --name $sync_task --state canceled
-#expect_succ test "$(echo "$OUTPUT" | jq -r length)" -eq $((count + 1))
-#expect_succ pulp task show --href "$task"
-#expect_succ test "$(echo "$OUTPUT" | jq -r '.state')" = "canceled"
+if [ "$(pulp debug has-plugin --name "core" --min-version "3.12.0")" = "true" ]
```

Comment:
```suggestion
if [ "$(pulp debug has-plugin --name "core" --min-version "3.12.0.dev")" = "true" ]
```

I think, this is the only way to test it ATM.

Reply:
Now if you'd rebase, we could get the test on the 3.12 container